[Finder] Update RealIteratorTestCase

flip111 opened the pull request:
| Q             | A
| ------------- | ---
| Branch?       | 2.8
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #27480
| License       | MIT
| Doc PR        | n/a

Makes the entire test directory empty instead of trying to delete particular files and directories. The old method failed when trying to remove a directory which was not empty.

flip111 committed:
[Finder] Update RealIteratorTestCase

nicolas-grekas commented on src/Symfony/Component/Finder/Tests/Iterator/RealIteratorTestCase.php:
```diff
@@ -69,11 +69,23 @@ public static function setUpBeforeClass()
-            if (DIRECTORY_SEPARATOR === $file[strlen($file) - 1]) {
-                @rmdir($file);
+        $paths = new \RecursiveIteratorIterator(
+                   new \RecursiveDirectoryIterator(
```
please use 4 spaces indentation

nicolas-grekas commented on src/Symfony/Component/Finder/Tests/Iterator/RealIteratorTestCase.php:
```diff
@@ -69,11 +69,23 @@ public static function setUpBeforeClass()
+                }
             } else {
-                @unlink($file);
+                unlink($path);
```
I'd suggest to keep the silencing operators (and add it above)

flip111 commented:
My current patch is based on master, should i open a new PR on 2.8 or what should i do about this?

gautierderuette committed:
[Validator] Remove BOM in some xlf files

nicolas-grekas committed:
bug #27630 [Validator][Form] Remove BOM in some xlf files (gautierderuette)

This PR was submitted for the 3.4 branch but it was merged into the 2.8 branch instead (closes #27630).

Discussion
----------

[Validator][Form] Remove BOM in some xlf files

| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   |
| Fixed tickets | #...
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

I removed first blank space from some xml files

It caused this error during cache:clear
[ERROR 4] Start tag expected, '<' not found (in n/a - line 1, column 1)

Commits
-------

0bc53d66c0 [Validator] Remove BOM in some xlf files

nicolas-grekas committed:
[Routing] remove unneeded dev dep on doctrine/common

nicolas-grekas commented:
Thank you @flip111.
